sites/skeleton.dev/package.json

```diff
@@ -12,10 +12,12 @@
 	},
 	"devDependencies": {
```

Comment:
Please undo any changes to this and the `pnpm-lock.yaml` files, these changes nothing of value.